src/test/java/tests/LoginTest.java

```diff
@@ -23,4 +25,21 @@ public void tc001_UserCanLogIntoRailwayWithValidUsernameAndPassword() {
         LogHelper.info("Verify that user can log into Railway with valid username and password");
         Assert.assertEquals(actual, "Welcome to Safe Railway", "Welcome message is not display like expected result");
     }
+
+    @Test(description = "User can't login with blank Username textbox")
+    public void tc02_UserCantLoginWithBlankUsernameTextbox() {
+        LogHelper.info("Click tab login");
+        loginPage.clickLoginTab();
+
+        LogHelper.info("Doesn't type any words into Username textbox but enter valid information into Password textbox" +
```

Comment:
You can make it simple by: "Login with empty username"

Reply:
fixed